scs2-session-visualizer-jfx/src/main/java/us/ihmc/scs2/sessionVisualizer/jfx/session/log/VideoViewer.java

```diff
@@ -228,7 +231,67 @@ protected void layoutChildren()
 
    public void update()
    {
-      FrameData currentFrameData = reader.pollCurrentFrame();
+      if (reader.getClass().equals(MagewellVideoDataReader.class))
```

Comment:
This is ugly, but allows support for the old and new logs, maybe there is a better way to do this @PotatoPeeler3000 @SylvainBertrand